<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>968</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>431</width>
      <height>459</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="input_params">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="horizontalSpacing">
      <number>5</number>
     </property>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="analysis_type">
         <property name="text">
          <string>Statistical analysis type:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="analysis_type_combo">
         <item>
          <property name="text">
           <string>Choose Test Type . . .</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Independent T-test</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dependent T-test</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>One Sample T-test</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Anova</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="in_location_label">
       <property name="cursor">
        <cursorShape>IBeamCursor</cursorShape>
       </property>
       <property name="text">
        <string>No file chosen</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="grp_names">
       <property name="text">
        <string>Column(s) with behavior data (behavior names):</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="grp_names_2">
       <property name="text">
        <string>Column(s) with group data (group numbers/names):</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPlainTextEdit" name="grp_names_text_2"/>
     </item>
     <item row="9" column="0">
      <widget class="QPlainTextEdit" name="ind_var_names_text"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="ind_var_names">
       <property name="text">
        <string>Column(s) with behavior start/stop time data:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QPlainTextEdit" name="grp_names_text"/>
     </item>
     <item row="10" column="0">
      <widget class="QPushButton" name="in_location_button">
       <property name="text">
        <string>Choose input file location:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTabWidget" name="images">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>471</width>
      <height>551</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Image 1</string>
     </attribute>
     <widget class="QLabel" name="display_pngs">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>461</width>
        <height>511</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Image 2</string>
     </attribute>
     <widget class="QLabel" name="display_pngs_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>461</width>
        <height>521</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>570</y>
      <width>431</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="save_graph_options">
     <item row="2" column="0">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="save_all_images">
       <property name="text">
        <string>Save all images</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="out_ppt_location_label">
       <property name="cursor">
        <cursorShape>IBeamCursor</cursorShape>
       </property>
       <property name="text">
        <string>No path chosen</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="out_ppt_location_button">
       <property name="text">
        <string>Choose ppt save location:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="save_one_image">
       <property name="text">
        <string>Save this image only</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="out_png_location_button">
       <property name="text">
        <string>Choose image save location:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="out_png_location_label">
       <property name="cursor">
        <cursorShape>IBeamCursor</cursorShape>
       </property>
       <property name="text">
        <string>No path chosen</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="save_ppt">
       <property name="text">
        <string>Save all images to power point</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>968</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
